src/utils/invoice.ts

```diff
@@ -23,6 +23,12 @@ export const liquidTestnetPrefix = "liquidtestnet:";
 
 export const maxExpiryHours = 24;
 
+const bolt11Prefixes = {
+    mainnet: "lnbc",
```

Comment:
i think its just main, looked it up in the config

````
export const network: string = "main";
```

Reply:
It used to be, now its `mainnet`